clusterloader2/testing/load/configmap.yaml

```diff
@@ -6,7 +6,29 @@ metadata:
 immutable: true
 {{end}}
 data:
-  data.yaml: |-
-    a: 1
-    b: 2
-    c: 3
+  all-queries: |
```

Comment:
Please add comment that this is for dns tests + add a comment that other use cases can be added here next to the all-queries file.

Reply:
done